conda/cli/helpers.py

```diff
@@ -180,7 +178,7 @@ def add_parser_json(p: ArgumentParser) -> _ArgumentGroup:
     )
     output_and_prompt_options.add_argument(
         "--console",
-        default=DEFAULT_CONSOLE_REPORTER_BACKEND,
+        default=NULL,
```

Comment:
Fixing a mistake from the last pull request to `feature/reporter-backends` (#14150)